lib/pg/text_encoder.rb

```diff
@@ -1,6 +1,7 @@
 # -*- ruby -*-
 
 require 'json'
+require 'ipaddr'
```

Comment:
It is not necessary, though it may be useful for documentation purposes.  I can remove the ipaddr require or add one for bigdecimal in the other pull request depending on what the consensus is.